### ISSUE TITLE
import does not work with Jython from the command line

### ISSUE THREAD

**Issue by pganuza** [RaiMan/SikuliX-2014#151]
Hello, and thanks for your work.

When I try to execute my test from the command line I get the following output:
```
E:\PDI\Cloud\QAVDC\Sikuli\Portal_Autoservicio> E:\PDI\Cloud\SikuliX\runsikulix.cmd -r Login.sikuli
+++ running this Java
java version "1.8.0_51"
Java(TM) SE Runtime Environment (build 1.8.0_51-b16)
Java HotSpot(TM) 64-Bit Server VM (build 25.51-b03, mixed mode)
+++ trying to run SikuliX
+++ using: -Xms64M -Xmx512M -Dfile.encoding=UTF-8 -Dsikuli.FromCommandLine -jar
E:\PDI\Cloud\SikuliX\sikulix.jar -r Login.sikuli
[error] script [ E:\PDI\Cloud\QAVDC\Sikuli\Portal_Autoservicio\Login.sikuli ] stopped with error in line 1
[error] ImportError ( No module named config )
```
However it works wonderfully when I execute it from SikuliX IDE.

conf.py file is in Login.sikuli folder and usually that is enough for python, shouldn't it be found?

**Comment by RaiMan** [comment]
are you using the latest 1.1.0 build?

run as
...runsikulix.cmd -d 3 -r Login.sikuli

to get more debug output.

**Comment by pganuza** [comment]
I used sikulixsetup-1.1.0-20150709.231110-123-forsetup.jar
```
E:\PDI\Cloud\SikuliX>runsikulix.cmd -d3 -r ..\QAVDC\Sikuli\Portal_Autoservicio\Login.sikuli\
+++ running this Java
java version "1.8.0_51"
Java(TM) SE Runtime Environment (build 1.8.0_51-b16)
Java HotSpot(TM) 64-Bit Server VM (build 25.51-b03, mixed mode)
+++ trying to run SikuliX
+++ using: -Xms64M -Xmx512M -Dfile.encoding=UTF-8 -Dsikuli.FromCommandLine -jar
E:\PDI\Cloud\SikuliX\sikulix.jar -d3 -r ..\QAVDC\Sikuli\Portal_Autoservicio\Login.sikuli\
[debug] Runner: commandline: -d3 -r ..\QAVDC\Sikuli\Portal_Autoservicio\Login.sikuli\
[debug] --- Sikuli parameters ---
[debug] 1: -d3
[debug] 2: -r
[debug] 3: ..\QAVDC\Sikuli\Portal_Autoservicio\Login.sikuli\
[debug] ScriptingSupport: givenScriptName: ..\QAVDC\Sikuli\Portal_Autoservicio\Login.sikuli\
[debug] ScriptingSupport: Trying to run script: E:\PDI\Cloud\QAVDC\Sikuli\Portal_Autoservicio\Login.sikuli\Login.py
[debug] ScriptingSupport: initScriptingSupport: enter
[debug] JythonSupport: init: starting
[debug] JythonSupport: init: success
[debug] JythonSupport: ***** Jython sys.path
 0: E:\PDI\Cloud\SikuliX\Lib
 1: E:\PDI\Cloud\SikuliX\sikulix.jar\Lib
 2: __classpath__
 3: __pyclasspath__/
[debug] JythonSupport: ***** Jython sys.path end
[debug] Jython: sikuli: Sikuli: starting init
[debug] RunTimeIDE: loadLib: VisionProxy.dll
[debug] Mouse: init
[debug] KeyBoard: init
[debug] RunTimeIDE: loadLib: WinUtil.dll
[debug] Jython: requested to use as default region: R[0,0 1366x768]@S(0)
[debug] Jython: sikuli: Sikuli: ending init
[debug] JythonScriptRunner: running Jython 2.7.0
[debug] ScriptingSupport: initScriptingSupport: added: jython
[debug] ScriptingSupport: initScriptingSupport: exit with defaultrunner: jython(py)
[debug] ImagePath: new BundlePath: /E:/PDI/Cloud/QAVDC/Sikuli/Portal_Autoservicio/Login.sikuli/
[debug] Jython: requested to use as default region: R[0,0 1366x768]@S(0)
[debug] JythonScriptRunner: runPython: running script: E:/PDI/Cloud/QAVDC/Sikuli/Portal_Autoservicio/Login.sikuli
[error] script [ E:\PDI\Cloud\QAVDC\Sikuli\Portal_Autoservicio\Login.sikuli ] stopped with error in line 1
[error] ImportError ( No module named config )
[debug] RunTimeIDE: final cleanup
[debug] FileManager: deleteFileOrFolder: C:\Users\TCPsi\AppData\Local\Temp\Sikulix_894043306
```

**Comment by pganuza** [comment]
It is the same with sikulixsetup-1.1.0-20150723.231118-137-forsetup.jar
```
E:\PDI\Cloud\SikuliX>runsikulix.cmd -d3 -r ..\QAVDC\Sikuli\Portal_Autoservicio\Login.sikuli\
+++ running this Java
java version "1.8.0_51"
Java(TM) SE Runtime Environment (build 1.8.0_51-b16)
Java HotSpot(TM) 64-Bit Server VM (build 25.51-b03, mixed mode)
+++ trying to run SikuliX
+++ using: -Xms64M -Xmx512M -Dfile.encoding=UTF-8 -Dsikuli.FromCommandLine -jar
E:\PDI\Cloud\SikuliX\sikulix.jar -d3 -r ..\QAVDC\Sikuli\Portal_Autoservicio\Login.sikuli\
[debug] Runner: commandline: -d3 -r ..\QAVDC\Sikuli\Portal_Autoservicio\Login.sikuli\
[debug] --- Sikuli parameters ---
[debug] 1: -d3
[debug] 2: -r
[debug] 3: ..\QAVDC\Sikuli\Portal_Autoservicio\Login.sikuli\
[debug] ScriptingSupport: givenScriptName: ..\QAVDC\Sikuli\Portal_Autoservicio\Login.sikuli\
[debug] ScriptingSupport: Trying to run script: E:\PDI\Cloud\QAVDC\Sikuli\Portal_Autoserviio\Login.sikuli\Login.py
[debug] ScriptingSupport: initScriptingSupport: enter
[debug] JythonSupport: init: starting
[debug] JythonSupport: init: success
[debug] JythonSupport: ***** Jython sys.path
 0: E:\PDI\Cloud\SikuliX\Lib
 1: E:\PDI\Cloud\SikuliX\sikulix.jar\Lib
 2: __classpath__
 3: __pyclasspath__/
[debug] JythonSupport: ***** Jython sys.path end
[debug] Jython: sikuli: Sikuli: starting init
[debug] RunTimeIDE: loadLib: VisionProxy.dll
[debug] Mouse: init
[debug] KeyBoard: init
[debug] RunTimeIDE: loadLib: WinUtil.dll
[debug] Jython: requested to use as default region: R[0,0 1366x768]@S(0)
[debug] Jython: sikuli: Sikuli: ending init
[debug] JythonScriptRunner: running Jython 2.7.0
[debug] ScriptingSupport: initScriptingSupport: added: jython
[debug] ScriptingSupport: initScriptingSupport: exit with defaultrunner: jython (py)
[debug] ImagePath: new BundlePath: /E:/PDI/Cloud/QAVDC/Sikuli/Portal_Autoservicio/Login.sikuli/
[debug] Jython: requested to use as default region: R[0,0 1366x768]@S(0)
[debug] JythonScriptRunner: runPython: running script: E:/PDI/Cloud/QAVDC/Sikuli/Portal_Autoservicio/Login.sikuli
[error] script [ E:\PDI\Cloud\QAVDC\Sikuli\Portal_Autoservicio\Login.sikuli ] stopped with error in line 1
[error] ImportError ( No module named conf )
[debug] RunTimeIDE: final cleanup
[debug] FileManager: deleteFileOrFolder: C:\Users\TCPsi\AppData\Local\Temp\Sikulix_1830672221
```

**Comment by RaiMan** [comment]
ok, you have to wait until mid of august, when I am back from vacation (cannot test it here ;-)

**Comment by pganuza** [comment]
No problem, relax and enjoy your vacation!

**Comment by kevin01523** [comment]
any updates for this?

**Comment by RaiMan** [comment]
Uuups, overseen - sorry.

Already tried with the final 1.1.0?
I will check in parallel.

**Comment by kevin01523** [comment]
yes tried on final this only happens on command line on ide no problem

**Comment by RaiMan** [comment]
Ok, this is a bug:
the bundlePath is not on sys.path when running a script from command line and hence .py cannot be found when stored in the bundle.

bug report:
https://bugs.launchpad.net/sikuli/+bug/1504901

Until fixed in version 1.1.1, use this hack (works in IDE (does nothing) and on command line):
# before the first import
import org.sikuli.util.JythonHelper
JythonHelper.get().addSysPath(getBundlePath())
